app/models/with_user_association_extension.rb

```diff
@@ -0,0 +1,16 @@
+module WithUserAssociationExtension
+  def create_with_user(attributes = {}, user)
```

Comment:
Style/OptionalArguments: Optional arguments should appear at the end of the argument list. (https://github.com/bbatsov/ruby-style-guide#optional-arguments)